packages/z/zed/pspec_x86_64.xml

```diff
@@ -0,0 +1,41 @@
+<PISI>
+    <Source>
+        <Name>zed</Name>
+        <Homepage>https://zed.dev/</Homepage>
+        <Packager>
+            <Name>Joey Riches</Name>
+            <Email>josephriches@gmail.com</Email>
+        </Packager>
+        <License>AGPL-3.0-or-later</License>
+        <License>Apache-2.0</License>
+        <License>GPL-3.0-or-later</License>
+        <PartOf>editor</PartOf>
+        <Summary xml:lang="en">Code at the speed of thought</Summary>
+        <Description xml:lang="en">Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. It&apos;s also open source.
+</Description>
+        <Archive type="binary" sha1sum="79eb0752a961b8e0d15c77d298c97498fbc89c5a">https://sources.getsol.us/README.Solus</Archive>
+    </Source>
+    <Package>
+        <Name>zed</Name>
+        <Summary xml:lang="en">Code at the speed of thought</Summary>
+        <Description xml:lang="en">Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. It&apos;s also open source.
+</Description>
+        <PartOf>editor</PartOf>
+        <Files>
+            <Path fileType="executable">/usr/bin/zed</Path>
```

Comment:
Please rename the binary to `Zed` to avoid conflicting with the `zed` binary from ZFS (which I'm aware we don't package but I'd prefer to avoid the conflict anyway if possible).